grammars/gfm.cson

```diff
@@ -655,7 +655,7 @@
     ]
   }
   {
-    'begin': '^\\s*([`~]{3,})\\s*(?i:(adoc|asciidoc|asciidoctor|asc|))\\s*$'
+    'begin': '^\\s*([`~]{3,})\\s*(?i:(adoc|asciidoc|asciidoctor|asc))\\s*$'
```

Comment:
@50Wliu Good grief. I only just noticed this, too:

~~~
([`~]{3,})
~~~

That'll wrongly match this as the start of a code-block:

~~~
~`~` asciidoc
~~~